components/places/Cargo.toml

```diff
@@ -32,6 +31,9 @@ dogear = "0.4"
 interrupt-support = { path = "../support/interrupt" }
 error-support = { path = "../support/error" }
 sync-guid = { path = "../support/guid", features = ["rusqlite_support", "random"]}
+thiserror = "1.0"
+anyhow = "1.0"
+backtrace = "0.3"
```

Comment:
Nit (which will probably cause a bit of churn): We should probably just be re-exporting backtrace from error-support, since it's a bit weird that you have to have the dep when no code directly depends on it.

something like:

```rust
/// Re-export of the `backtrace` crate for use in macros and
/// to ensure the needed version is kept in sync in dependents.
pub use backtrace;
```

And inside the macros you would do e.g. `$crate::backtrace::Backtrace` to refer to the backtrace object.

Reply:
Ooo that is smart!! Added it in my latest push 🕺